package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "graknlabs-grpc-protocol",
+  "name": "graknlabs-protocol",
```

Comment:
shouldn't this be "grakn-protocol" tho

Reply:
It doesn't actually need to be - but it does make sense to keep consistent, so I've updated it to `grakn-protocol`.